OpenRA.Mods.Common/Traits/CustomSelectionSize.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Special case trait for invisible, unselectable actors like bridge huts.",
+	"Gives actor targetable area for special cases like C4 and engineer repair.",
```

Comment:
How about
"Special case trait for unselectable actors that need do define targetable area bounds for special cases like C4, engineer repair and tooltips. Examples: bridge huts and crates."